src/forms/formUtils.tsx

```diff
@@ -320,6 +320,25 @@ export const renderDatacenterItem: ItemRenderer<DatacenterObject> = (
   );
 };
 
+export const renderSiteItem: ItemRenderer<DatacenterObject> = (
+  site: DatacenterObject,
+  { handleClick, modifiers, query }
+) => {
+  if (!modifiers.matchesPredicate) {
+    return null;
+  }
+  const text = site.name + " (" + site.abbreviation + ")";
+  const siteType = site.is_storage ? "Offline Storage" : "Datacenter";
```

Comment:
sites show up differently in this drop down so we can indicate what's offline storage & what's datacenter. bonus - the abbreviation match still gets highlighted and queried even though it's moved to the text and not the label! (you can't search by the sitetype text but the user should know the name of the site they're moving to so i think that's fine)